playwright/utils/test_setup.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  chromium,
+  firefox,
+  webkit,
+  ChromiumBrowser,
+  FirefoxBrowser,
+  WebKitBrowser,
+  BrowserType,
+  Page,
+  BrowserContext,
+} from 'playwright'
+import { assert } from '@gnosis.pm/dex-js'
+export let browser: ChromiumBrowser | FirefoxBrowser | WebKitBrowser
+export let context: BrowserContext
+export let page: Page
+
+export const baseURL = process.env.BASE_URL || 'http://localhost:8080'
+
+const availableBrowsers: Record<string, BrowserType<ChromiumBrowser | FirefoxBrowser | WebKitBrowser>> = {
+  chromium,
+  firefox,
+  webkit,
+}
+
+const browserFlag = process.env.BROWSER
+
+assert(
+  !browserFlag || browserFlag in availableBrowsers,
+  `unsupported environment variable BROWSER=${browserFlag}. Use ${Object.keys(availableBrowsers).join(', ')}`,
+)
+
+const browserType = browserFlag ? availableBrowsers[browserFlag] : chromium
+
+const executablePath = process.env.EXECUTABLE_PATH
+
+// PWDEBUG is playwright specific flag
+// sets up window.playwright and other stuff
+const isDebug = !!process.env.PWDEBUG
+let slowMo: number | undefined
+
+// timeout to have time to account for async nature of browser actions
+jest.setTimeout(30000)
+if (isDebug) {
+  // slow down playright actions
+  slowMo = 2000
```

Comment:
Maybe add a const on top, and explain that artificially delay the test to visually see what is going on